libs/data-mapper/src/lib/models/WebviewMessages.ts

```diff
@@ -32,7 +34,7 @@ export type MessageToVsix =
       data: string;
     }
   | {
-      command: 'webviewLoaded' | 'readLocalSchemaFileOptions' | 'readLocalCustomXsltFileOptions';
+      command: 'initialize' | 'webviewLoaded' | 'readLocalSchemaFileOptions' | 'readLocalCustomXsltFileOptions';
     }
```

Comment:
@ccastrotrejo Should we use ExtensionCommands constants here when this is data-mapper not related to vs-code?

Reply:
Yeah, I would say yes. The purpose its the same the command between the vscode and react